src/dataflow/components/dataflow-program-toolbar.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import { BaseComponent, IBaseProps } from "./dataflow-base";
+import { observer, inject } from "mobx-react";
+import { NodeTypes } from "../utilities/node";
+
+import "./dataflow-program-toolbar.sass";
+
+interface IProps extends IBaseProps {
+  onNodeCreateClick: (type: string) => void;
+  onDeleteClick: () => void;
+  onResetClick: () => void;
+  onClearClick: () => void;
+}
+interface IState {}
+
+@inject("stores")
+@observer
+export class DataflowProgramToolbar extends BaseComponent<IProps, IState> {
```

Comment:
This is a simple class that appears not to need state, access to stores, or MobX/MobX State Tree. Simplify it by making it a functional component that doesn't have anything it doesn't need.

Reply:
Agreed.  Fixed.